packages/staking/src/features/portfolio-drift/isPortfolioDrifted.ts

```diff
@@ -20,8 +22,19 @@ const getPortfolioTotalPercentageDrift = (portfolio: CurrentPortfolioStakePool[]
   );
 };
 
+const isSavedPercentagePresent = (
+  currentPortfolio: CurrentPortfolioStakePool[]
+): currentPortfolio is (CurrentPortfolioStakePool & { savedIntegerPercentage: number })[] =>
+  currentPortfolio.some(({ savedIntegerPercentage }) => !savedIntegerPercentage);
```

Comment:
I guess this should actually be the inverse: 

```suggestion
  currentPortfolio.every(({ savedIntegerPercentage }) => savedIntegerPercentage);
```

also, I'd consider calling the function `hisPortfolioWithSavedPercentages` which IMHO aligns better with its purpose/logic

Reply:
Nice catch! Thanks for spotting this! I might have missed this case while testing 😅